chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -154,7 +154,8 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 
       if node[:nova][:libvirt_type] == "kvm"
         if node.platform_version.to_f >= 12.0
-          package "qemu-kvm"
+          package "qemu-kvm" if node[:architecture] == "x86_64"
+          package "qemu-ppc" if node[:architecture] =~ "ppc64le"
```

Comment:
@k0da are we using suggests, recommends or requires for this? I'm not sure if chef-client will respect let zypper use suggests/recommends...